src/hooks/useDataBuffer.ts

```diff
@@ -29,9 +29,13 @@ export const useDataBuffer = (
 
   const sendDataOutThrottled = useMemo(
     () =>
-      throttle(() => {
-        setDataOut(dataIn.current);
-      }, delay),
+      throttle(
+        () => {
+          setDataOut(dataIn.current);
+        },
+        delay,
+        { leading: false, trailing: true }
```

Comment:
The root cause of the misbehaviour of the first line is from lodash that was introduced in useDataBuffer.
when leading set to true, the first call is executed immediately (which we do not want).
when trailing is set to true, the subsequent calls within the throttle delay are ignored until the delay period elapses.

Reply:
Commit title was not changed